test/razorpay/test_payment.rb

```diff
@@ -4,8 +4,11 @@ module Razorpay
   # Tests for Razorpay::Payment
   class RazorpayPaymentTest < Minitest::Test
     def setup
-      @payment_id = 'pay_FFX5FdEYx8jPwA'
-
+      @payment_id = 'fake_payment_id'
+      @downtime_id = 'fake_downtime_id'
+      @card_id = 'card_7EZLhWkDt05n7V'
+      @transfer_id = 'trf_J0FrZYPql4riDx'
+      @refund_id = 'fake_refund_id'
```

Comment:
Change the Refund ID format you used in the Refund Module(fake_refund.json) or update in `test_payments_fetch_refunds `test case